javagen/src/main/java/com/azure/autorest/util/ProxyMethodExampleWrapper.java

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.autorest.util;
+
+public class ProxyMethodExampleWrapper {
+
+    private final Object xmsExample;
+    private final String operationId;
+
+    public ProxyMethodExampleWrapper(Object xmsExample, String operationId) {
+        this.xmsExample = xmsExample;
+        this.operationId = operationId;
+    }
+
+    public Object getXmsExample() {
+        return xmsExample;
+    }
+
+    public String getOperationId() {
+        return operationId;
```

Comment:
The operationId is already in ProxyMethod?

Reply:
Fine for this.

Though you also need to add the example name to this structure?

Reply:
Added.